content/en/about/feature-stages/index.md

```diff
@@ -16,17 +16,29 @@ within the project, not to the project as a whole. Here is a high level descript
+| Support               | No guarantees on backward compatibility. The feature may be removed at any time without notice.                                                                                                                                                                                                                                                                                                                                                                            | No guarantees on backward compatibility. The feature may be removed at any time without notice.                                                                                                                                                                                                                                                                                                                                                    | The overall feature will not be dropped, though details may change.                                                                                                                                                                                                                                                                                                                                                                                                                         | The feature will appear in released software for many subsequent versions.                                                                 |
```

Comment:
Here you mention the feature won't be dropped (only column with no drop), but we talk about deprecation on the next row.

Reply:
I think that's an issue we could file in istio/community and the TSC can update that wording.